### DISCUSSION TITLE
playback modifier

### DISCUSSION THREAD

**Discussion by DeceptedPar** [DEMOLITIONDON96/Demolition-Engine#118] [Ideas (Other)]
I saw in the github you done a lofi/hifi thing, you didn't have to do it that way
someone had already coded a playback speed modifier
https://github.com/Starmapo/FNF-PsychEngine/tree/playbackrate
`this was also in the actions tab for psych engine`

**Comment by ThatOneSillyGuyDon** [comment]
This sounds like a really neat idea
